### ISSUE TITLE
footballdatabase.com

### ISSUE THREAD

**Issue by kelson42** [openzim/zim-requests#68]
_From @amirh123123 on March 9, 2018 16:53_

hi please make zim file for footbaldatabase.com

_Copied from original issue: openzim/mwoffliner#216_

**Comment by kelson42** [comment]
unreachable web site to me.

**Comment by kelson42** [comment]
_From @amirh123123 on March 19, 2018 6:53_

The #1 Football / Soccer World Club Ranking Site on the Internet.

(CC BY-SA 4.0) FootballDatabase.co

**Comment by RavanJAltaie** [comment]
@amirh123123 
Requested
https://farm.openzim.org/recipes/footballdatabase.com

**Comment by RavanJAltaie** [comment]
Succeeded.

**Comment by Popolechien** [comment]
Only captured the first page.

**Comment by benoit74** [comment]
Reopening, ZIM is still not ready.

Configuration was conflicting (`url` is not covered by `include` rule, `exclude` seems to be garbage):

<img width="874" alt="image" src="https://github.com/openzim/zim-requests/assets/7102089/78f07ca9-1086-4839-8537-99ef57e68c79">

I fixed the recipe and requested again.

Nota: ZIM name will have to be fixed as well

**Comment by benoit74** [comment]
Could you please have a look at dev library, it looks like file is mostly OK? Only metadata (ZIM name at least) needs to be fixed.

**Comment by Popolechien** [comment]
Found a formatting issue at https://dev.library.kiwix.org/viewer#footballdatabase_2024-06/footballdatabase.com/ranking/europe/1 and then clicking on pretty much any link on that page returns a missing page (check out the URL, it does not look like it is encoded properly)

**Comment by benoit74** [comment]
This is an upstream issue, I've opened a ticket to investigate

**Comment by benoit74** [comment]
Upstream bug closed, trying again

**Comment by benoit74** [comment]
@Popolechien can you please have a look at https://dev.library.kiwix.org/#lang=eng&q=football+database, it seems ok to me

**Comment by Popolechien** [comment]
Yup, looks good to me too. Moved the recipe to prod, will close issue when ready.

**Comment by Popolechien** [comment]
Moved to prod successfully. There's a search bar at top right that ought to be disabled, but no biggie.
